src/shared/alias-mappings.ts

```diff
@@ -19,6 +31,10 @@ export const mappings: Record<AliasName, string[]> = {
 		"sf1626",
 		"sf1671",
 	],
+	[AliasName.Y1P1]: ["sf1671", "dd1337", "dd1390", "da1600"],
+	[AliasName.Y1P2]: ["sf1624", "dd1338", "dd1337", "dd1390", "da1600"],
+	[AliasName.Y1P3]: ["sf1625", "dd1338", "dd1390", "dh1623"],
+	[AliasName.Y1P4]: ["sf1626", "dd1396", "dd1349", "dd1390"],
 	[AliasName.YEAR2]: [
```

Comment:
Just a thought. Does it not make more sense to have a `course` object contacting the `id` and the `period/year`? It feels like that's were the information belongs. Wdyt? @Hampfh

Reply:
I talked to Noah and Douglas and it seems like it will be easier for admins to access and maintain the data with this format